projects/batfish/src/org/batfish/grammar/cisco/CiscoLexer.g4

```diff
@@ -2391,6 +2412,11 @@ GROUP_URL
    'group-url'
 ;
 
+GSHUT
```

Comment:
GSHUT is a valid keyword. See http://www.cisco.com/c/en/us/td/docs/ios-xml/ios/iproute_bgp/configuration/xe-3s/irg-xe-3s-book/irg-grace-shut.pdf